spring-cloud-generator/scripts/generate-steps.sh

```diff
@@ -116,15 +142,17 @@ function bazel_build_all(){
 #
 # $1 - path-to-pom-file (e.g. path/to/pom.xml)
 # $2 - starter-artifact-id (e.g. google-cloud-accessapproval-spring-starter)
-function add_module_to_pom () {
-  xmllint --debug --nsclean --xpath  "//*[local-name()='module']/text()" $1 \
-    | sort | uniq | grep -q $2 || module_list_is_empty=1
+function add_module_to_aggregator_pom() {
+  path_to_pom=$1
```

Comment:
Addressing a previous suggestion in https://github.com/GoogleCloudPlatform/spring-cloud-gcp/pull/1956#discussion_r1231548395: updated helper functions to consistently declare descriptive variable names for the arguments they expect ($1, $2, ...)